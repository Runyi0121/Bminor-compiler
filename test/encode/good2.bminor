"0xPP \0x0a 0xzz \0xFF 0xqq \0x0d \0x08"