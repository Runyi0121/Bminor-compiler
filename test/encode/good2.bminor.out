"0xPP \n 0xzz \0xFF 0xqq \r \b"
